syncapi/streams/stream_pdu.go

```diff
@@ -423,8 +423,12 @@ func (p *PDUStreamProvider) addIgnoredUsersToFilter(ctx context.Context, req *ty
 		return err
 	}
 	req.IgnoredUsers = *ignores
+	var userList []string = nil
```

Comment:
`userList := make([]string, 0, len(ignores.List))` to reduce the number of re-allocations.